ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/HiveInsertExchange4JoinRule.java

```diff
@@ -56,20 +57,28 @@
  */
 public class HiveInsertExchange4JoinRule extends RelOptRule {
 
-  protected static transient final Logger LOG = LoggerFactory
-      .getLogger(HiveInsertExchange4JoinRule.class);
+  protected static final Logger LOG = LoggerFactory.getLogger(HiveInsertExchange4JoinRule.class);
```

Comment:
The logger does not seem to be used so I guess we can drop it entirely along with the relevant imports.

Reply:
Removed